components/Header.js

```diff
@@ -0,0 +1,37 @@
+import Image from "next/image";
+import styled from "styled-components";
+import menue from "../public/images/menü_btn.jpg";
+
+export default function Header({ mapShown }) {
+  return (
+    <>
```

Comment:
You don't need the fragments here. `<StyledHeader>` wraps around the return statement